lib/commands/context.js

```diff
@@ -87,6 +87,13 @@ extensions.getNewRemoteDebugger = async function getNewRemoteDebugger () {
   }, this.isRealDevice());
 };
 
+/**
+ * Set context
+ *
+ * @param {?string} name - The name of context to set. It could be 'null' as NATIVE_WIN.
+ * @param {callback} callback The callback. (It is not called in this method)
```

Comment:
Can we remove this callback with removing `_.noop` in `await this.setContext(name, _.noop, skipReadyCheck)`?

- ios-driver
```
lib/commands/context.js:commands.setContext = async function setContext (name, callback, skipReadyCheck) {
lib/commands/context.js:      await this.setContext(latestWindow);
test/e2e/testapp/context-specs.js:    await driver.setContext(contexts[0]);
test/e2e/testapp/context-specs.js:    await driver.setContext(null);
test/e2e/testapp/context-specs.js:    await driver.setContext(contexts[0]);
test/e2e/testapp/context-specs.js:    await driver.setContext('NATIVE_APP');
test/e2e/testapp/context-specs.js:    await B.resolve(driver.setContext('WEBVIEW_42'))
```

- this repository
```
$ git grep "setContext"
lib/commands/context.js:commands.setContext = async function setContext (name, callback, skipReadyCheck) {
lib/commands/context.js:    await this.setContext(name, _.noop, skipReadyCheck);
Binary file test/assets/TouchIDExample.app/TouchIDExample matches
```

Reply:
@imurchie